File: JAVA/Ruster exercise/com/Ruster/ARL.java

/**
 *
 * @author JUAN
 */
package com.Ruster;

public class ARL {
    private double calculateARL;

    public ARL(double salary) {
        this.calculateARL = salary * 0.052; // 5.2%
    }

    public double getCalculateARL() {
        return this.calculateARL;
    }
}



  

File: PHP/Ruster exercise/Ruster/ARL.php
<?php
class ARL extends Person {
    private float $calculateARL;

    public function __construct(float $salary) {
        $this->salary = $salary;
        $this->calculateARL = $this->salary * 0.052;
    }

    public function getCalculateARL(): float {
        return $this->calculateARL;
    }


    public function getDescription(): string {}
    public function getNetSalary(): float {}
}



